BridgeCareApp/BridgeCare/Services/SummaryReport/SummaryReportBridgeData.cs

```diff
@@ -0,0 +1,361 @@
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) > 10 ? "N" : yearData.SubD;
+            worksheet.Cells[row, ++coln].Value = Convert.ToInt32(familyId) < 11 ? "N" : yearData.CulvD;
+            worksheet.Cells[row, ++coln].Value = yearData.MinC;
+            worksheet.Cells[row, ++coln].Value = yearData.SD;
+            if (yearData.Year != 0)
+            {
+                worksheet.Cells[row, ++coln].Value = yearData.Project;
+                worksheet.Cells[row, ++coln].Value = yearData.Cost;
+            }
+            // Empty column
+            coln++;
+            return coln;
+        }
+
+        private static List<SimulationDataModel> GetSimulationDataModels(DataTable simulationDataTable, List<int> simulationYears, IQueryable<ReportProjectCost> projectCostModels)
+        {
+            var simulationDMs = new List<SimulationDataModel>();
+            var projectCostsList = projectCostModels.ToList();
+            foreach (DataRow simulationRow in simulationDataTable.Rows)
+            {
+                var simulationDM = CreatePrevYearSimulationMdel(simulationRow);
+                var projectCostEntry = projectCostsList.Where(pc => pc.SECTIONID == Convert.ToUInt32(simulationRow["SECTIONID"])).FirstOrDefault();
+                AddAllYearsData(simulationRow, simulationYears, projectCostEntry, simulationDM);
+                simulationDMs.Add(simulationDM);
+            }
+
+            return simulationDMs;
+        }
+
+        private static void AddAllYearsData(DataRow simulationRow, List<int> simulationYears, ReportProjectCost projectCostEntry, SimulationDataModel simulationDM)
+        {
+            var yearsDMs = new List<YearsData>();
+            foreach (int year in simulationYears)
+            {
+                yearsDMs.Add(AddYearsData(simulationRow, projectCostEntry, year));
+            }
+            simulationDM.YearsData.AddRange(yearsDMs.OrderBy(y => y.Year).ToList());
+        }
+
+        private static SimulationDataModel CreatePrevYearSimulationMdel(DataRow simulationRow)
+        {
+            YearsData yearsData = AddYearsData(simulationRow, null, 0);
+            return new SimulationDataModel
+            {
+                YearsData = new List<YearsData>() { yearsData },
+                SectionId = Convert.ToInt32(simulationRow["SECTIONID"])
+            };
+        }
+
+        private static YearsData AddYearsData(DataRow simulationRow, ReportProjectCost projectCostEntry, int year)
+        {
+            var yearsData = new YearsData
+            {
+                Deck = simulationRow["DECK_SEEDED_" + year].ToString(),
+                Super = simulationRow["SUP_SEEDED_" + year].ToString(),
+                Sub = simulationRow["SUB_SEEDED_" + year].ToString(),
+                Culv = simulationRow["CULV_SEEDED_" + year].ToString(),
+                DeckD = simulationRow["DECK_DURATION_N_" + year].ToString(),
```

Comment:
Dont abbreviate. Unless the Excel output is requiring it. If so, leave a comment.

Reply:
Yes, these are columns from excel report, kept same in code to get it where data belongs.